alpine/distributionscanner.go

```diff
@@ -105,8 +113,15 @@ func (ds *DistributionScanner) Scan(ctx context.Context, l *claircore.Layer) ([]
 		log.Debug().Msg("didn't find an os-release or issue file")
 		return nil, nil
 	}
-	for _, buff := range files {
-		dist := ds.parse(buff)
+	// Always check the os-release file first.
```

Comment:
why this change?

Reply:
Mostly because checking the files in random order (because of map iteration order being random) really bugs me, conceptually. It's not, strictly speaking, necessary.